exercises/raindrops/test/test_raindrops.c

```diff
@@ -11,99 +12,106 @@ void tearDown(void)
 {
 }
 
+void test_convert_drops(int drops, char *expected)
```

Comment:
We are trying to change how helper functions are named in test files, it would be great if you could remove the 'test_' prefix here

Reply:
👍